timescaledb/how-to-guides/multinode-timescaledb/multinode-maintenance.md

```diff
@@ -23,6 +26,36 @@ $$;
 SELECT add_job('data_node_maintenance', '5m');
 ```
 
+## Maintaining statistics for distributed hypertables
```

Comment:
We should try and only use gerund headings for Procedures. I think in this case we can ditch the verb altogether, since it's a concept:

```suggestion
## Statistics for distributed hypertables
```

Reply:
Done